projects/valtimo/user-interface/src/lib/components/table/table.component.scss

```diff
@@ -42,8 +42,7 @@
 }
 
 .v-table__body {
-  box-shadow: var(--v-table-body-box-shadow);
-  border-radius: var(--v-border-radius-md);
+  border: var(--v-table-body-box-border);
```

Comment:
Excuse me for not mentioning this in previous PR comments. 

You have now defined the same border property twice. One time under input component `border: 1px solid var(--v-input-border-color);` and one time under `border: var(--v-table-body-box-border);` for the table component. 

You should define the border property once generally and then use them in the component specific tokens. In this way, the design remains consistent in the future. This system is also explained [here[(https://valtimo.gitbook.io/valtimo-core-docs/reference/user-interface/design-philosophy).

so it should be something like:
/* borders */
--v-border-medium: 1px solid var(--v-color-grey-3);

...

--v-table-body-border: var(--v-border-medium);

...

--v-input-border: var(--v-border-medium);